config/config.gb.default.yaml

```diff
@@ -140,7 +221,7 @@ low_carbon_register:
 solving:
   options:
     clip_p_max_pu: 1.e-2
-    load_shedding: false
+    load_shedding: 6000 #£/MWh - based on `value_of_lost_load` from (https://www.nationalgrid.com/sites/default/files/documents/Long-term%20Market%20and%20Network%20Constraint%20Modelling.pdf)
```

Comment:
Looks like we just need to be able to switch this separately for the unconstrained and constrained run. For unconstrained it would be for debugging, for constrained it would be an expected addition